jujugui/static/gui/src/app/app.js

```diff
@@ -469,7 +464,7 @@ YUI.add('juju-gui', function(Y) {
      * @param {Object} cfg Application configuration data.
      * @param {Object} env The environment instance.
      */
-    _init: function(cfg, env) {
+    _init: function(cfg, env, controllerAPI) {
```

Comment:
I renamed all of the `env` to `modelAPI` but left `this.env` alone as there are many places throughout the application that use this naming. It can definitely be done as a follow-up though